install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml

```diff
@@ -673,6 +710,8 @@ spec:
       {{- end }}
       {{- end }}
       volumes:
+      - name: tmp
```

Comment:
nit: Might be nice to add a comment to highlight that this is declared to share the path between the config initContainer and the main container.